lib/syskit/scripts/log_runtime_archive.rb

```diff
@@ -18,12 +18,21 @@ def self.exit_on_failure?
            type: :numeric, default: 600, desc: "polling period in seconds"
     option :max_size,
            type: :numeric, default: 10_000, desc: "max log size in MB"
+    option :free_space_low_limit,
+           type: :numeric, default: 1_000, desc: "start deleting files if free space is \
+            below this threshold"
+    option :free_space_delete_until,
+           type: :numeric, default: 10_000, desc: "stop deleting files if free space is \
```

Comment:
use the constants you have defined in LogRuntimeArchive (actually true for the other parameters)